sheets/3/exercise-4.tex

```diff
@@ -1,10 +1,35 @@
-\excercise{Collections}
+\excercise{Operation parameters}
 \subsection*{a)}
-	Passe die Main Methode an die Aufgabe an.
-\subsections*{b)}
+	Passe die Main Methode an die Aufgabe an. Verwende \lstinline{Sheet3Task4} und \lstinline{Sheet3Task4Verifier}.
+\subsection*{b)}
 	Schreibe eine Methode die Neo einen Schritt nach vorne gehen lässt.
-\subsections*{c)}
-	- Schreibe Methoden die Neo 2 Schritte nach vorne gehen lässt.
-	- Schreibe Methoden die Neo 3 Schritte nach vorne gehen lässt.
+\subsection*{c)}
+	- Schreibe Methoden die Neo 2 Schritte nach vorne gehen lässt.\\
+	- Schreibe Methoden die Neo 3 Schritte nach vorne gehen lässt.\\
 	- Schreibe Methoden die Neo 4 Schritte nach vorne gehen lässt.
-\subsection*{d)}
\ No newline at end of file
+\subsection*{d)}
+	\begin{Infobox}[Operations Parameter]
+		Wenn du eine Operation schreiben möchtest die je nach Situation mit verschiedenen Variablen die selbe Operation ausführt kannst du der Methode einen Parameter übergeben. Bisher haben wir der Methode immer den Neo übergeben der alle Operationen ausführen sollte, man kann aber jede art von Objekt als Parameter übergeben.
+	
+	\begin{lstlisting}
+		public void printNumbers(int from, int to){
```

Comment:
Bitte in Beispielen nach möglichkeit die Objekt Wrapper verwenden (hier Integer)